app/scripts/lib/ppom/types.ts

```diff
@@ -0,0 +1,8 @@
+export type SecurityAlertResponse = {
+  description?: string;
+  features?: string[];
+  providerRequestsCount?: Record<string, number>;
+  reason: string;
+  result_type: string;
+  securityAlertId?: string;
+};
```

Comment:
Looks redundant of this: https://github.com/MetaMask/metamask-extension/blob/07f1eb412350d32231cadfda00d1347acb28749c/app/scripts/lib/transaction/util.ts#L32